pom.xml

```diff
@@ -163,15 +164,15 @@
         </dependency>
 
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.13.2</version>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-params</artifactId>
+            <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.junit.jupiter</groupId>
-            <artifactId>junit-jupiter-params</artifactId>
-            <version>5.9.2</version>
+            <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
Very small one, but could you add this one above the `junit-jupiter-params`, please? Just because this is the main dependency, so it makes sense to be the first one that's listed.